packages/json-schema/src/build-schema.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2013,2018. All Rights Reserved.
+// Node module: @loopback/json-schema
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {SchemaObject, MapObject} from '@loopback/openapi-spec';
+import {ModelMetadataHelper, PropertyDefinition} from '@loopback/repository';
+import {includes} from 'lodash';
+
+/**
+ * Type definition for a JSON Schema Definition
+ * Currently, objects of type JSONDefinition can also be cast as a SchemaObject,
+ * a property of OpenAPI-v2's specification
+ */
+export type JsonDefinition = {
```

Comment:
I don't mind maintaining our own copy of TypeScript interface/type definition for JSON Schema, but it still makes me wonder - isn't there any npm package we could leverage for that?